core/templates/services/math-interactions.service.ts

```diff
@@ -524,6 +524,42 @@ export class MathInteractionsService {
   // The input terms to this function should be the terms split by '+'/'-'
   // from an expression.
   termsMatch(term1: string, term2: string): boolean {
+    let getConstants = function(expression: string): string[] {
+      let constants = expression.match(/(\d*\.\d+)|(\d+)/g);
+      if (constants !== null) {
+        return constants;
+      }
+      return [];
+    };
+
+    let partsMatch = function(part1: string, part2: string): boolean {
```

Comment:
Start with a verb: doPartsMatch.

Can you add a comment defining a "part", though?

Reply:
Done.